packages/govuk-frontend/tasks/build/release.unit.test.mjs

```diff
@@ -126,7 +126,7 @@ describe('dist/', () => {
 
     it('should contain relative paths to sources', () => {
       expect(sourcemap.sources).toContain(
-        '../packages/govuk-frontend/src/govuk/all.mjs'
+        '../packages/govuk-frontend/src/govuk/init.mjs'
```

Comment:
**note** `sourcemap.sources` lists many other modules, which was already the case alongside `all.mjs`, so we can just swap to another one of our files being imported.